adage/analyze/api.py

```diff
@@ -5,7 +5,7 @@
 from organisms.api import OrganismResource
 from genes.api import GeneResource
 from tastypie import fields, http
-from tastypie.resources import Resource, ModelResource
+from tastypie.resources import Resource, ModelResource, convert_post_to_VERB
```

Comment:
This is an interesting approach. I am surprised that tastypie doc never mentioned it. Can you add at least one test case in `tests.py` to confirm that it works? (I forgot to do that when adding `ExpressionValueResource` class. My bad.)